hub/util/from_tfds.py

```diff
@@ -0,0 +1,43 @@
+import tensorflow  # type: ignore
+import tensorflow_datasets as tfds  # type: ignore
+from tqdm import tqdm  # type: ignore
+
+from hub import Dataset
+
+
+def from_tfds_to_path(
+    tfds_dataset_name: str, split: str, hub_ds_path: str, batch_size: int = 100
+):
+    """Converts the tfds dataset with name `tfds_dataset_name` into a hub dataset and saves it at `hub_ds_path`
+    Args:
+        tfds_dataset_name (str): Name of tfds dataset.You can see a list of all tfds datasets here:
+            https://www.tensorflow.org/datasets/catalog/overview
+        split (str) : Enum for dataset splits used for loading a certain tfds dataset split
```

Comment:
[Not an enum.](https://www.tensorflow.org/datasets/splits)

Reply:
It's an enum for tfds_datasets. We just pass this. The type is "Str" for us.

Reply:
@imshashank please see the link I provided. The syntax they support is very powerful, and definitely not enumerative. Please at least link it, as it appears we're forwarding the parameter and supporting it all.